datafusion/optimizer/src/utils.rs

```diff
@@ -66,6 +66,16 @@ pub fn optimize_children(
     }
 }
 
+/// Returns true if all columns in col_refs are in `schema_cols`
+///
+/// Note: can't use `HashSet::intersect` here because they have different types.
+pub(crate) fn has_all_refs(
```

Comment:
maybe we can rename it to better reflect `if all columns in col_refs are in `schema_cols` ?

Reply:
in d4346f384b5a843997a9c0fcb37878b5007a024e